api/src/main/scala/dcos/metronome/api/v0/controllers/ScheduledJobSpecController.scala

```diff
@@ -12,18 +11,17 @@ import mesosphere.marathon.metrics.Metrics
 import mesosphere.marathon.plugin.auth.{ Authenticator, Authorizer, CreateRunSpec, UpdateRunSpec }
 import play.api.libs.functional.syntax._
 import play.api.libs.json._
-import play.api.mvc.{ AnyContent, BodyParser }
+import play.api.mvc.ControllerComponents
 
 import scala.concurrent.ExecutionContext
 
-class ScheduledJobSpecController(
-  jobSpecService:        JobSpecService,
-  val metrics:           Metrics,
-  val authenticator:     Authenticator,
-  val authorizer:        Authorizer,
-  val config:            ApiConfig,
-  val mat:               Materializer,
-  val defaultBodyParser: BodyParser[AnyContent])(implicit ec: ExecutionContext) extends Authorization {
+class ScheduledJobSpecController(cc: ControllerComponents)(
+  implicit
+  val ec: ExecutionContext, jobSpecService: JobSpecService, metrics: Metrics,
```

Comment:
I prefer to either have all on 1 or 2 lines or all on separate lines... not a combo.   In this case.. all separate lines as it was previously.